core/src/entities/scans/scan.rs

```diff
@@ -0,0 +1,147 @@
+use crate::entities::packages::FindingProviderKind;
+use crate::entities::sboms::SbomProviderKind;
+use crate::Error;
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+use std::collections::HashMap;
+
+/// A [Scan] is a value type that contains the results of an enrichment cycle where the SBOM is
+/// assessed for vulnerability [Findings].
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+#[skip_serializing_none]
+pub struct Scan {
```

Comment:
@qtpeters  I'm not sure where we ended up with this in terms of `Job` vs `Batch` vs `Scan` vs `Sync`.  All are relevant and correct at different levels of abstraction.